crates/cranelift/src/func_environ.rs

```diff
@@ -461,6 +461,12 @@ mod typed_continuation_helpers {
         pointer_type: ir::Type,
     }
 
+    #[derive(Copy, Clone)]
+    pub struct Vmctx {
```

Comment:
Nit
```suggestion
    pub struct VMCtx {
```

Reply:
Went with `VMContext`. After all, the other structs in this module are already named after the objects they are used to access.